### PR TITLE
Update brainvision.py

### PR DESCRIPTION
**Pull request by jnvandermeer**
#### Reference issue
The Brain Vision export from the EEGo system (the sports caps) do not import due to a typo by whoever programmed the Brain Vision Export on the EEGo development team.

#### What does this implement/fix?
To make io.brainvision more 'robust' by adding their incorrect header information (containing "BrainVision" instead of "Brain Vision"), so MNE doesn't throw an error. I tested on my system but recommend to test it first on another system too.

### CODE REVIEW

File: mne/io/brainvision/brainvision.py [hunk 1]
```diff
@@ -478,8 +483,10 @@ def _check_hdr_version(header):
 def _check_mrk_version(header):
     """Check the marker version."""
     tags = ['Brain Vision Data Exchange Marker File, Version 1.0',
-            'Brain Vision Data Exchange Marker File Version 1.0',
-            'Brain Vision Data Exchange Marker File, Version 2.0']
+            'BrainVision Data Exchange Marker File, Version 1.0',
+            'BrainVision Data Exchange Marker File Version 1.0',
```
Comment (cbrnr) [inline mne/io/brainvision/brainvision.py]: Here you removed an existing valid tag, namely `'Brain Vision Data Exchange Marker File Version 1.0'` - if you add this back the test should pass.

Comment (jnvandermeer) [reply mne/io/brainvision/brainvision.py]: Dear Clemens and Joan, thank you, I'm using MNE only for a couple of months so I'm not too experienced yet in how to go about this.

I also think a regex could actually be a bit better - or maybe just a header.endswith('1.0') or .endswith('2.0') to determine which version header is used. I haven't done many pull requests, so I'm now not sure what I should at this time and if my mistake was resolved by Joan already. Should I generate a new pull request, work in this request, or rather not do anything at this point? Sorry for the newbie questions.

**Comment by agramfort** [comment]
2 things

- @jnvandermeer you should not send PRs from your master branch
- this should work but a regexp would be nicer and should simplify the checks

**Comment by massich** [comment]
@jnvandermeer no worries. In general terms, read our contributing guide. we
should also have some github instructions that shows the work flow we use
to develop mne. (there it should be explained why not to use master). If
while you are reading the documentation, you find anything confusing, not
detailed enough or whatever, then please DO open issues or submit PR. We
would very much appreciate it. Ping me if you need anything. (you can also
share your doubts in our gitter)

Usually we don't push to other people PRs. But I did here just in case that
you desapear (it happens) so that it would still be merged. Plus I didn't
see that @cbrnr already pointed you what to change. Otherwise I would had
not. So that you could fix it.

At this point we can do the regex in this PR (you would have to pull the
changes to your master) or you can let this pr as is, see if at least 2
developers agree on merge it as is. And then open a PR that refactors the
code using a regex. (you would actually need only another contributer to
approve the PR 'cos I'm fine with enter this one like this and do a
subsequent PR)

To sum up. Thanks for contributing.

+1

**Comment by cbrnr** [comment]
I'm OK with moving the regex into a new PR. However, Travis fails with (1) a MPL-related issue (which should be fixed in #5660), and (2) with a PEP8-related error. @jnvandermeer do you have time to fix the PEP8 errors?

**Comment by massich** [comment]
The pep8 was also unrelated. I'll double check and merge

**Comment by massich** [comment]
The PeP8 has to do with #5657. merging.

@jnvandermeer Thx! and welcome to the family of MNE contributors.

**Comment by jnvandermeer** [comment]
Thank you very much! I'll get back to reading up on why-not-to-use master, and the regex, once I return from holidays.
